build/configs/rtl8730e/rtl8730e_make_bin.sh

```diff
@@ -83,10 +83,14 @@ $GNUUTL/prepend_header.sh $BINDIR/bl1_sram.bin  __ca32_bl1_sram_start__  $BINDIR
 $GNUUTL/prepend_header.sh $BINDIR/bl1.bin  __ca32_bl1_dram_start__  $BINDIR/target_img2.map
 $GNUUTL/prepend_header.sh $BINDIR/fip.bin  __ca32_fip_dram_start__  $BINDIR/target_img2.map
 
-$GNUUTL/prepend_header.sh $GNUUTL/rtk_ext_flash_data.bin  __flash_ext_data_start__  $BINDIR/target_img2.map
-cp $GNUUTL/rtk_ext_flash_data_prepend.bin $BINDIR/rtk_ext_flash_data.bin
-rm $GNUUTL/rtk_ext_flash_data.bin
-rm $GNUUTL/rtk_ext_flash_data_prepend.bin
+if [ "$ext_flash_support" = "1" ];then
```

Comment:
@jeongarmy Please check here~

Reply:
According to the logic here, if we enter this condition, means the binary creation is successful. Else, it should print the logs for condition `ext_flash_support == 0`.
I tried on flat and loadable config, it is working as expected.
If you think we still need some logs, then I shall add it here.